framework/auth/core.py

```diff
@@ -1004,6 +1022,26 @@ def display_absolute_url(self):
     def deep_url(self):
         return '/profile/{}/'.format(self._primary_key)
 
+    @property
+    def get_unconfirmed_emails(self):
```

Comment:
Not sure where my previous comments went, but this needs to be renamed . Properties should be named like attributes (nouns rather than actions). Also, the docstring needs to be updated.